parm/config/config.fcst

```diff
@@ -147,9 +194,23 @@ elif [ $imp_physics -eq 6 ]; then # WSM6
 
 elif [ $imp_physics -eq 8 ]; then # Thompson
     export ncld=2
-    export FIELD_TABLE="$HOMEgfs/parm/parm_fv3diag/field_table_thompson${tbf}"
+    export FIELD_TABLE="$HOMEgfs/parm/parm_fv3diag/field_table_thompson_noaero_tke"
```

Comment:
What is "noaero" in this context? No MERRA2? No GOCART? Something else? It this is really the field table we want?

Reply:
Yes this is the field table we want.  I believe it means no aerosols in the Thompson MP